Recipe/style.css

```diff
@@ -0,0 +1,104 @@
+
+.TOPtitleAndParagraph {
+    text-align: center;
+    font-family: 'Bitter', serif;
+}
+
+
+/* This does not work */
+@media (max-width: 1080px) {
+    #paragraph {
+        color: pink;
+    }
+}
+
+
+#h1Title {
+    color: #6B4E37;
+    
+}
+
+
+.LEFTimgPCIngre {
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+}
+
+
+img {
+    margin: 0 auto;
+    display: block;
+    width: 50%;
+}
+
+
+/* added a media query for the image, image gets smaller when the screen is minamized */
+@media (max-width: 1080px){
+    img {
+        width: 35%;
+    }
+}
+
+
+.test{
+    display: flex;
+}
+
+
+#photoCredit {
+    text-align: center;
+    font-family: 'Bitter', serif;
+}
+
+
+.Ingredients {
+    background-color: rgb(245, 171, 81);
+    width: 45%;
+    display: inline-block;
+    margin: 0 auto;
+    font-family: 'Bitter', serif;
+    text-align: center;
+    color: #6B4E37
+}
+
+
+.Instructions {
+    background-color: rgb(245, 171, 81);
+    width: 45%;
+    display: inline-block;
+    margin: 0 auto;
+    font-family: 'Bitter', serif;
+    text-align: center;
+    color: #6B4E37
+}
```

Comment:
You see how these two classes are pretty much the same lines of code. 
```
{
    background-color: rgb(245, 171, 81);
    width: 45%;
    display: inline-block;
    margin: 0 auto;
    font-family: 'Bitter', serif;
    text-align: center;
    color: #6B4E37
}
```
You can consolidate these into one class instead so you don't have duplicates.  Then use the id to differentiate between the two.

For example...
In style.css
```
.listContainer {
    background-color: rgb(245, 171, 81);
    width: 45%;
    display: inline-block;
    margin: 0 auto;
    font-family: 'Bitter', serif;
    text-align: center;
    color: #6B4E37
}
```
In index.html
```
<div id="ingredients" class="listContainer">
          <h3>Ingredients</h3>
          <ul>
            <li>1 1/4 cups all purpose flour</li>
            <li>1 cup yellow corn meal</li>
            <li>2/3 cup granulated sugar</li>
            <li>1 teaspoon salt</li>
            <li>1 tablespoon baking powder</li>
            <li>1 1/4 cup unsweetened almond milk</li>
            <li>1/3 cup canola oil</li>
          </ul>
        </div>

        <div id="instructions" class="listContainer">
          <h3>Instructions</h3>
          <ol id="Instructions">
            <li>Preheat the oven to 400 degrees F and lightly grease a 8 x 8 pan, or a 9 inch round cake pan.</li>
            <li> In a large bowl, combine the flour, cornmeal, sugar, salt and baking powder and stir.</li>
            <li> Now pour in the almond milk and canola oil. Stir until well combined. Pour batter into prepared pan.</li>
            <li> Bake for 20-25 minutes, until a toothpick inserted into the center comes out clean.</li>
          </ol>
        </div>
```